src/wurf/context_msg_resolver.py

```diff
@@ -26,9 +24,10 @@ def resolve(self):
 
         :return: The path as a string.
         """
+        
 
         self.ctx.start_msg('{} {}'.format(
-            self.method, self.dependency.name))
+            self.dependency.resolver_method, self.dependency.name))
```

Comment:
Alright, I think it would be good to find a nicer way to print this information. Should we also do a separate PR for that?